### ISSUE TITLE
modified model and the output becomes different!

### ISSUE THREAD

**Issue by jingpengw** [sovrasov/flops-counter.pytorch#32]
After using this package, my model output is changed a little bit!
My model is 3D U-Net, and the input is a 3D image patch, the output is a 3d patch.

It took me some time to debug this and finally found that the problem is from using this package!

**Comment by jingpengw** [comment]
sorry, not this package's problem. it is from random initialization of dynamic batch norm.
